test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Remove **ALL** comments in this file. They are auto-generated by scaffolding command. Of course, you don't need them 'cause you can totally understand the codes related to them. Let the codes tell your story!